bulk_grades/views.py

```diff
@@ -69,7 +69,8 @@ def get(self, request, course_id, *args, **kwargs):
         filename = self.get_export_filename(course_id)
 
         response = StreamingHttpResponse(iterator, content_type='text/csv')
-        response['Content-Disposition'] = f'attachment; filename="{filename}"'
+        response_content = 'attachment;'
```

Comment:
Any reference for this change ?

Reply:
This statement was raising pylint error `Multiple statements on one line (semicolon) (E702)`